utilix/batchq.py

```diff
@@ -58,6 +59,7 @@ def submit_job(jobstring,
                container='xenonnt-development.simg',
                bind=('/dali', '/project2', os.path.dirname(TMPDIR)),
                cpus_per_task=1,
+               hours=24,
```

Comment:
I think the default should be the default behavior of SLURM. Can you maybe change this to `None` and then don't add the SBATCH option if `hours` is None? 

https://slurm.schedmd.com/sbatch.html